todolist.html

```diff
@@ -32,5 +32,57 @@
                 <i class='icon icon--left fa fa-trash-o'></i> Delete All
             </button>
         </div>
+        <div>
+            <p>Add Task</p>
+            <div>
+                <label>Name</label>
+                <div>100</div>
+                <input type='text'>
+                <small>Maximum of 100 characters.</small>
+            </div>
+            <div>
+                <label>Expected date of completion</label>
+                <input type='text'>
+                <small></small>
+            </div>
+            <div>
+                <label>Status</label>
+                <select>
+                    <option value='to-do'>To-Do</option>
+                    <option value='in-progress'>In Progress</option>
+                    <option value='done'>Done</option>
+                </select>
+            </div>
+            <div>
+                <label>Tag</label>
+                <div>30</div>
+                <input type='text'>
+                <small>Tags help to group tasks together. Maximum of 30 characters.</small>
+            </div>
+            <div>
+                <label>Colour of Tag</label>
+                <select>
+                    <option value='blue'>Blue</option>
+                    <option value='green'>Green</option>
+                    <option value='lgreen'>Light Green</option>
+                    <option value='orange'>Orange</option>
+                    <option value='lorange'>Light Orange</option>
+                </select>
+            </div>
+            <div>
+                <label>Description</label>
+                <div>250</div>
+                <textarea></textarea>
+                <small>Maximum of 250 characters.</small>
+            </div>
+            <div>
+                <button type='button'>
+                    Save Task
```

Comment:
How come the text for this button is spaced and indented instead of being left in one line alike all the other lines of code such as "<small>Tags help to group tasks together. Maximum of 30 characters.</small>" and "<option value='lorange'>Light Orange</option>"

Reply:
We were told this is the proper style for buttons